### PR TITLE
Migrate legacy DAOs script

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: legacy-dao-params-generator.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+          voteParams {
+            ${voteParasQuery}
+          }
+          rewarder
+        }
+      }
+    }
+  }`
+  try {
+    let { data } = (await axios.post('https://api.thegraph.com/subgraphs/name/daostack/v39_4', { query })).data
```
Comment (orenyodfat) [inline legacy-dao-params-generator.js:80]: make this env var.

File: migrate-dao.js [hunk 2]
```diff
@@ -199,7 +199,7 @@ async function migrateDAO ({ arcVersion, web3, spinner, confirm, opts, migration
     founders.map(({ reputation }) => web3.utils.toWei(reputation !== undefined ? reputation.toString() : '0'))
   ]
 
-  const initFoundersBatchSize = 20
+  const initFoundersBatchSize = 100
```
Comment (orenyodfat) [inline migrate-dao.js:202]: does this change related to the transition script?

File: legacy-dao-params-generator.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
+const axios = require('axios')
+axios.defaults.timeout = 30000
+
+const Web3 = require('web3')
+const web3 = new Web3()
+const BN = require('bn.js')
+
+async function getLegacyDAOParameters (daoId) {
```
Comment (orenyodfat) [inline legacy-dao-params-generator.js:8]: from where come the daoId ? is that env var ?

File: migrate-dao.js (hunk 2 above)
Comment (ben-kaufman) [reply migrate-dao.js:202]: Partly so, because the transition requires usually a lot of founders, and I had to modify this part anyway to allow that.

**Pull request merged**